.github/workflows/build.yml

```diff
@@ -185,14 +185,14 @@ jobs:
       - name: Test all files (sage -t --all --long)
         if: (success() || failure()) && steps.build.outcome == 'success'
         run: |
-          ../sage -python -m pip install coverage
-          ../sage -python -m coverage run ./bin/sage-runtests --all --long -p2 --random-seed=286735480429121101562228604801325644303
-        working-directory: ./worktree-image/src
+          ./sage -python -m pip install coverage
+          ./sage -python -m coverage run ./src/bin/sage-runtests --all --long -p2 --format github --random-seed=286735480429121101562228604801325644303
+        working-directory: ./worktree-image
```

Comment:
That change doesn't seem to work, and is also unrelated to the actual goal of the PR.

Reply:
> unrelated to the actual goal of the PR.

It's necessary for the printed paths to be correct. Otherwise annotations don't work.

Reply:
> That change doesn't seem to work

Please, Tobias, include the necessary information when you say that something doesn't work.

Reply:
I assume you meant that the next step, "Prepare coverage results", failed. That's fixed now.